extensions/ql-vscode/src/vscode-tests/no-workspace/remote-queries/factories/cli.ts

```diff
@@ -0,0 +1,29 @@
+import { CodeQLCliServer } from '../../../../cli';
+
+export function createMockCliServer(
+  sandbox: sinon.SinonSandbox,
+  mockOperations: Record<string, any[]> = {}
+): CodeQLCliServer {
+  if (mockOperations == {}) {
```

Comment:
I don't think this will work since objects are reference types, so even `{} == {}` will be `false`. Instead, you can add the defaults as part of the function arguments or make the default `undefined` (although then you will also need to change the argument type).

However, I don't think we are even using these operations. So, would it also work to simply have an empty default (i.e. no operations are mocked)?